.github/workflows/verify_newproject_script.yml

```
@@ -40,6 +40,11 @@ jobs:
           sdk install kscript $kscriptVersion
           echo $PATH >> $GITHUB_PATH
 
-      - name: Verify generating new project from template
+      - name: Verify generating new project from template-xml
         working-directory: scripts
-        run: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
+        run: kscript new_project.kts package-name=co.myproject.example app-name="My Project" template=xml
```

Why this app-name doesn't have `XML` at the end like `My Project Compose`?
Besides, I think adding XML in front of `Project` makes more sense to me.

```suggestion
        run: kscript new_project.kts package-name=co.myxmlproject.example app-name="My XML Project" template=xml
```